mcs/tools/resx2sr/resx2sr.cs

```diff
@@ -111,13 +119,24 @@ static void GenerateFile (List<Tuple<string, string, string>> txtStrings, CmdOpt
 			str.WriteLine ("//");
 			str.WriteLine ();
 
-			str.WriteLine ("partial class SR");
+			int nsIdx = className.LastIndexOf('.');
```

Comment:
Would be great to keep the parens formatting same for the changes

Reply:
```suggestion
			int nsIdx = className.LastIndexOf ('.');
```